configdialog.h

```diff
@@ -81,6 +83,41 @@ class PaletteCustomizeDialog {
 	PaletteGroupRepository m_group_repo;
 };
 
+class PalettesImport {
+public:
+	static PalettesImport *parse_palettes_string(const std::wstring &import_string);
+	std::optional<std::wstring> get_error_string(size_t max_errors);
+	size_t get_valid_palettes_amount();
+	std::wstring get_valid_palette_names(size_t max_length);
+	void import_palettes();
+
+private:
+	std::vector<std::wstring> palettes_name;
+	std::vector<std::vector<std::wstring>> palettes_raw_colors;
+	std::vector<unsigned int> palettes_has_errors;
+	bool import_string_had_terminator;
+
+	const static inline std::wstring valid_chars = L"qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM0123456789-_!?:";	// Note: Spaces are valid characters but not included in this string
```

Comment:
It's sort of weird that `(` and `)` are not valid characters even though a duplicate palette import creates a palette name with for example `(2)` in it. I think that parens can be added to the valid characters list.

Reply:
That is... a bit of an oversight. But yeah, we can probably add the parentheses to the valid characters.